### DISCUSSION TITLE
GoPro Hero11 Mini - USB Power Trigger & Auto Launch WiFi???

### DISCUSSION THREAD

**Discussion by madgit23** [gopro/labs#463] [Q&A]
Hi!

I'm new here after spending many hours searching the web for an answer...

I'm looking to purchase a Hero11 Mini, but first need to make sure that I'll be able to get it to function how I need. Basically, I'll be using the camera with LivePreview (in the GoPro Quick app) and an iPad for a real-time visual on part of a theatre stage that's obscured from line-of-sight.

I've seen that the Hero11 Mini can be set up to power on automatically using the USB Power Trigger (https://gopro.github.io/labs/control/usb/) which I'll need, as the camera will be left rigged up in a ceiling-height lighting rig. However, I'll also need the camera to automatically enable WiFi when it's powered on via the USB Power Trigger, so that I can connect the iPad to the camera via the Quick app. Is that possible and, if so, how would I do it??

Thank you in advance for any suggestions!

**Comment by dnewman-gpsw** [comment]
Labs features are bonuses, not guaranteed, so you shouldn't purchase a camera on needing Labs features. 

Most features can be enabled with boot and wake on power hacks, I haven't tried this setup.
